src/generate-source-info.ts

```diff
@@ -99,7 +99,7 @@ for (const [sourceTag, matchRule] of Object.entries(categories.sources)) {
               (i.itemCategoryHashes?.includes(ItemCategoryHashes.Weapon) ||
                 i.itemCategoryHashes?.includes(ItemCategoryHashes.Armor) ||
                 i.itemCategoryHashes?.includes(ItemCategoryHashes.Ghost) ||
-                i.itemCategoryHashes?.includes(ItemCategoryHashes.Shaders) ||
+                i.traitIds?.includes('item.shader') ||
```

Comment:
Bungie removed the Shaders ICH from all items -- we should do something to clearly mark unused ICHs as deprecated, perhaps through UNUSED_Shaders or `@deprecated`?

Reply:
Note that all trait ids changed, so it's just as likely that it's a bug that the shader itemCategoryHash (which is a fiction of the bnet api) is no longer granted to shaders, and should be reported + fixed 🤞🏻

Reply:
Thank you, I didn't consider that -- I agree a bug sounds the most likely explanation. Shaders seem to be uniquely affected https://github.com/Bungie-net/api/issues/1829